build.sbt

```diff
@@ -83,6 +83,7 @@ lazy val runtime = CrossProject("scalac-scoverage-runtime", file("scalac-scovera
       fork in Test := true
     )
     .jsSettings(
+      name := name.value + "_s" + crossProjectPlatform.value.identifier,
```

Comment:
Yea this won't work either actually...

Because we are doing a full cross release here, that means in the sbt plugin we also need to target it like so:

```scala
libraryDependencies +=
  "org.scoverage" %% "scalac-scoverage-plugin" % "1.4.3-SNAPSHOT" cross CrossVersion.full
```

When doing this. during the js tests it will then fail since it'll be looking for `_sjs1.0_2.12-1.4.3-SNAPSHOT`. It seems that doing the full cross release and the js stuff here together is really screwing with things.